packages/flutter/lib/src/widgets/scroll_simulation.dart

```diff
@@ -53,19 +62,21 @@ class BouncingScrollSimulation extends Simulation {
       // Taken from UIScrollView.decelerationRate (.normal = 0.998)
       // 0.998^1000 = ~0.135
       _frictionSimulation = FrictionSimulation(0.135, position, velocity);
-      final double finalX = _frictionSimulation.finalX;
+      final double finalX = _frictionSimulation!.finalX;
       if (velocity > 0.0 && finalX > trailingExtent) {
-        _springTime = _frictionSimulation.timeAtX(trailingExtent);
+        _springTime = _frictionSimulation!.timeAtX(trailingExtent);
         _springSimulation = _overscrollSimulation(
           trailingExtent,
-          math.min(_frictionSimulation.dx(_springTime), maxSpringTransferVelocity),
+          math.min(
+              _frictionSimulation!.dx(_springTime), maxSpringTransferVelocity),
         );
```

Comment:
Same here and elsewhere, please see the style guide for formatting guidance. The flutter repo does not use dart format.

```suggestion
          math.min(
            _frictionSimulation!.dx(_springTime),
            maxSpringTransferVelocity),
        );
```